client/beefy/src/metrics.rs

```diff
@@ -37,6 +37,7 @@ pub(crate) struct Metrics {
 }
 
 impl Metrics {
+	#![cfg_attr(test, allow(dead_code))]
```

Comment:
Why do need the `allow(dead_code)`? This is used when initializing the BEEFY worker.